### ISSUE TITLE
No exp folder 

### ISSUE THREAD

**Issue by shadsakib** [gsig/PyVideoResearch#28]
One of the steps to train the network says to duplicate and edit one of the experiment files under exp/ with appropriate parameters. I'm not sure where exp/ and the experiment files being referred to are.

**Comment by gsig** [comment]
The experiment templates are under baseline_exp/, you can then make a dir
called exp/ for your own experiments.

Hope that helps!

**Comment by shadsakib** [comment]
Thank you. 

One more thing, is it possible to test the models on random video clips? It's not clear to me how I can test it and I can't seem to find any instructions either.

**Comment by gsig** [comment]
There are experiment files for running the pretrained models on the test
sets under baseline_exp/. The "test set" is determined by the *_test.csv
file (see datasets/ for details on the data loaders, so it would be easiest
to just put your data on that format and choose your csv file in the
config.

For example, adding a row to Charades_v1_test.csv with a video name and
labels would test on that video.

Typically I create many different csv files to test on different types of
videos or labels etc

Hope that helps!

**Comment by shadsakib** [comment]
Thanks a lot. That was helpful.
